### PR TITLE
[release-4.13][manual] e2e: rte: backport local tests

### PR DESCRIPTION
**Pull request by ffromani**
move local exoporter tests in their own suite, backporting from 4.14

partial backport of #653

### CODE REVIEW

File: hack/run-test-e2e.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ export E2E_TOPOLOGY_MANAGER_POLICY="${E2E_TOPOLOGY_MANAGER_POLICY:-SingleNUMANod
 # --fail-fast: ginkgo will stop the suite right after the first spec failure
 # --flake-attempts: rerun the test if it fails
 # -requireSuite: fail if tests are not executed because of missing suite
-${BIN_DIR}/e2e-nrop-rte.test ${NO_COLOR} --ginkgo.v --ginkgo.timeout=5h --ginkgo.fail-fast --ginkgo.flake-attempts=2 --ginkgo.junit-report=${REPORT_DIR}/e2e.xml --ginkgo.skip='\[Disruptive\]|\[StateDirectories\]|\[NodeRefresh\]|\[local\]'
+${BIN_DIR}/e2e-nrop-rte.test ${NO_COLOR} --ginkgo.v --ginkgo.timeout=5h --ginkgo.fail-fast --ginkgo.flake-attempts=2 --ginkgo.junit-report=${REPORT_DIR}/e2e.xml --ginkgo.skip='\[Disruptive\]|\[StateDirectories\]|\[NodeRefresh\]|\[local\]' --ginkgo.focus='\[release\]'
```
Comment (shajmakh) [inline hack/run-test-e2e.sh:66]: I realize this is a manual backport, but I want to ensure I get the changes correctly. I couldn't find any rte test with the [release] label. am I missing something?

Comment (ffromani) [reply hack/run-test-e2e.sh:66]: we mean to consume the vendored rte tests: e.g. https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/main/test/e2e/rte/rte.go#L221

**Pull request merged**